pkg/storage/s3/getter.go

```diff
@@ -100,7 +100,7 @@ func (s *Storage) open(ctx context.Context, path string) (storage.SizeReadCloser
 	if err != nil {
 		return nil, errors.E(op, err)
 	}
-	var size int64
+	var size int64 = -1
```

Comment:
Given the comment above, would it be okay to revert this change and keep it at zero? This way all storage implementations have consistent behavior hopefully.